Library/Homebrew/manpages/brew.1.md

```diff
@@ -556,6 +556,14 @@ can take several different forms:
 
     *Note:* Homebrew will only try to print emoji on Lion or newer.
 
+  * HOMEBREW\_NO\_INSECURE\_REDIRECT:
+    If set, Homebrew will not permit redirects from secure download
+    protocols to insecure ones.
+
+    Whist ensuring your downloads are fully secure, this is likely
```

Comment:
Whoops. There's a typo there that I didn't spot, even after reading your comment. Apologies. Happy with "while" though, making the change now.